freedomofpress/securedrop
# Allow viewing of Tor browser functional test over VNC

## Status

Ready for review

## Description of Changes

Fixes #3718 

Changes proposed in this pull request:
* Does some re-configuration with the existing VNC helper function and uses Xvnc instead (quite frankly I had a lot of issues trying to get it working with `x11vnc` + `Xvfb` )
* Adds some helper commands 

## Testing

How should the reviewer test this PR?

If you want to test against a remote server:
* Start a staging server (either via the upgrade scenario or manually)
* If you didn't use `upgrade` scenario - fill out the `instance_information.json` and do the create_dev dance
* Start the tests `cd securedrop ;  ./bin/dev-shell ./bin/run-test tests/functional` 
* While those are running (once you start seeing test section) run `make func-vnc` )
* :tada:  you should see the tests running 

Of special note is that I did `NOT` run this under macOS yet so I'll need anyone with `macOS` *cough* *cough* @redshiftzero *cough* to take a look and make sure that `make func-vnc` call works

## Checklist

- [ ] How many woodchucks?
